opm/material/fluidsystems/blackoilpvt/WetGasPvt.hpp

```diff
@@ -557,16 +557,13 @@ class WetGasPvt
             const Evaluation& f = RvTable.eval(pSat, /*extrapolate=*/true) - Rv;
             const Evaluation& fPrime = RvTable.evalDerivative(pSat, /*extrapolate=*/true);
 
-            const Evaluation& delta = f/fPrime;
-
-            if (!Opm::isfinite(delta)) {
-                 std::stringstream errlog;
-                 errlog << "Obtain delta with non-finite value for Rv = " << Rv << " in the "
-                        << i << "th iteration during finding saturation pressure iteratively";
-                 OpmLog::problem("wetgas NaN delta value", errlog.str());
-                 OPM_THROW_NOLOG(NumericalProblem, errlog.str());
+            //If the derivative is zero Newton will not converge, so simply return our best guess instead
+            if (Toolbox::scalarValue(fPrime) == 0.0) {
+                return pSat;
```

Comment:
yeah, some versions produce it, some don't. I guess that if you set `-Wextra`, exact float comparisons will always be complained about, though.